atomicfu/src/jvmTest/kotlin/kotlinx/atomicfu/test/TraceToStringTest.kt

```diff
@@ -0,0 +1,22 @@
+package kotlinx.atomicfu.test
+
+import kotlinx.atomicfu.Trace
+import kotlinx.atomicfu.atomic
+import org.junit.Test
+import kotlin.test.assertEquals
+
+class TraceToStringTest {
+    private val aTrace = Trace { i, text -> "[$i: $text]" }
+    private val a = atomic(0, aTrace)
+
+    @Test
+    fun testTraceToStringAtomicInt() {
```

Comment:
👍 Very good. You should also add a simple test for manual `aTrace { ... }` calls.